css/components-list.css

```diff
@@ -0,0 +1,124 @@
+@import url("https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@400;700&display=swap");
+@font-face {
+  font-family: Jurassic Park;
+  src: url(JurassicPark-BL48.ttf);
+}
+
+:root {
+  --primary-color: #00919c;
+  --title-color: #cccccc;
+  --subtitle-color: #808080;
+  --card-bg-color: #27272785;
+  --background-color: #101212;
+}
+
+html,
+body,
+h1,
+p,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Jurassic Park;
+  letter-spacing: 1px;
+  background-color: var(--background-color);
+}
+
+.header-nav {
+  display: flex;
+  align-items: center;
+  flex-wrap: nowrap;
+  margin-top: 1rem;
+}
+
+.header-logo-link {
+  display: flex;
+  align-items: center;
+  flex-wrap: nowrap;
+  text-decoration: none;
+}
+
+.header-logo-img {
+  width: 48px;
+  height: 48px;
+  padding-left: 1rem;
+}
+
+.header-logo-name {
+  font-size: 2.5rem;
+  color: var(--title-color);
+  text-transform: lowercase;
+  padding-left: 6px;
+}
+
+.header-nav-links {
+  margin-left: auto;
+}
+
+.header-nav-links > nav > ul {
+  list-style: none;
+  display: flex;
+  gap: 1rem;
+  padding-right: 10px;
```

Comment:
use rem for 10 and above px

Reply:
Yeah, I did have a thought on this one. But this was an edge case so I went ahead with px.